bin/run-addon

```diff
@@ -79,6 +81,14 @@ while [[ -n "$1" ]] ; do
   esac
 done
 
+open="open $binary" # assumes Mac OS, availability of 'open' command
```

Comment:
This seems fine – defaulting to Linux and looking for `"$(uname)" = "Darwin" ` would also be fine.